src/python/pants/notes/2.3.x.md

```diff
@@ -0,0 +1,39 @@
+## 2.3.0.dev0 (Jan 10, 2021)
+
+### User API Changes
+
+* Remove support for pex_binary sources. ([#11445](https://github.com/pantsbuild/pants/pull/11445))
+
+* Remove custom targets for Pylint and MyPy. ([#11441](https://github.com/pantsbuild/pants/pull/11441))
+
+* Remove deprecated --pantsd-pailgun-quit-timeout option. ([#11440](https://github.com/pantsbuild/pants/pull/11440))
+
+* Remove DeprecatedRunTracker. ([#11439](https://github.com/pantsbuild/pants/pull/11439))
+
+* Remove deprecated PythonAwsLambda fields. ([#11438](https://github.com/pantsbuild/pants/pull/11438))
+
+* Remove deprecated v1 Subsystem facilities ([#11424](https://github.com/pantsbuild/pants/pull/11424))
+
+* Remove python workunits ([#11416](https://github.com/pantsbuild/pants/pull/11416))
+
+### Plugin API Changes
+
+* Remove support for 'Lite' request object hacks. ([#11444](https://github.com/pantsbuild/pants/pull/11444))
+
+* Remove support for docstring help. ([#11442](https://github.com/pantsbuild/pants/pull/11442))
+
+### Bug fixes
+
+* Add `--remote-cache-eager-fetch` to reduce unreliability ([#11396](https://github.com/pantsbuild/pants/pull/11396))
```

Comment:
Probably better in new feature, but this works too.

Reply:
As I understood it this was a stopgap bugfix that would be replaced by better handling with no special feature / configuration use needed going forward.

Reply:
That's how I viewed it at first too, but Benjy conjectured it may always be worth keeping if the performance is better. I removed the `help` message saying "we plan to deprecate this" after review feedback.